# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/base/PresShell.cpp
# Commit: f8ca02252dfc
# Full Hash: f8ca02252dfc48d7342703e6a635d510b9dc0b5d
# Author: Jonathan Kew <jkew@mozilla.com>
# Date: 2020-04-02 09:51:45
# Regressor Bug: 1619349
# File Overlap Count: 1
# Description:
#   Bug 1619349 - patch 1 - Record statistics about font-matching behavior in the presContext. r=jwatt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D67787
# ==============================================================================

diff -r 01a2a527aa1a -r f8ca02252dfc layout/base/PresShell.cpp
--- a/layout/base/PresShell.cpp	Wed Apr 01 21:42:11 2020 +0000
+++ b/layout/base/PresShell.cpp	Wed Apr 01 21:42:36 2020 +0000
@@ -10250,6 +10250,7 @@
       nsFontMetrics::Params params;
       params.language = nsGkAtoms::x_western;
       params.textPerf = aPresContext->GetTextPerfMetrics();
+      params.fontStats = nullptr;  // not interested here
       params.featureValueLookup = aPresContext->GetFontFeatureValuesLookup();
       RefPtr<nsFontMetrics> fm =
           aPresContext->DeviceContext()->GetMetricsFor(font, params);